README.md

```diff
@@ -25,9 +25,8 @@ To get a local copy up and running follow these simple example steps.
 
 - Run `git clone https://github.com/TimmyChan99/To_Do_list.git`
 - Go to the folder `cd To_Do_list`
-- Install packages run `npm install`
-- To run application using webpack-dev-server run `npm start`
-
+- Install packages: run `npm install`
+- To run application using webpack-dev-server: run `npm start`
 
 
 ## Authors
```

Comment:
- [ ] **Required:** To visitors especially recruiters see what your application looks like, ensure your screenshot is working correctly. At the moment, the link is broken.
![image](https://user-images.githubusercontent.com/27709832/149089476-cb4f07d2-4c2b-4353-b841-935326b0a902.png)

Reply:
I have checked and the screenshot works fine on my PC and other devices
![Screenshot 2022-01-12 094423](https://user-images.githubusercontent.com/92228303/149095949-66109fd9-bb4d-4cde-8f72-2ba81e8461aa.png)